tests/test_taxi.py

```diff
@@ -69,7 +67,7 @@ def test_car_is_register_in_admin(self):
 
 class GitignoreTests(TestCase):
     def test_gitignore_exist(self):
-        file_exists = os.path.exists('.gitignore')
+        file_exists = os.path.exists(".gitignore")
         assert file_exists
```

Comment:
The use of `assert` in Django tests is not recommended. Instead, use `self.assertTrue(file_exists)` to ensure consistency with Django's testing framework.